### PR TITLE
Improve scaling mysql increment synchronization

### PR DESCRIPTION
**Pull request by azexcy**
Changes proposed in this pull request:
- Improve MySQLClient reconnect at high concurrency
- Reduce json length, use normal length.
- Comments Code

### CODE REVIEW

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/client/MySQLClient.java [hunk 1]
```diff
@@ -294,12 +297,11 @@ public void channelInactive(final ChannelHandlerContext ctx) {
...
         public void exceptionCaught(final ChannelHandlerContext ctx, final Throwable cause) {
             running = false;
-            log.error("protocol resolution error", cause);
+            log.error("MySQLBinlogEventHandler: protocol resolution error", cause);
         }
         
         private void reconnect() {
-            reconnectTimes++;
-            log.info("reconnect mysql client, retryTimes={}", reconnectTimes);
+            log.info("reconnect mysql client, retryTimes={}", reconnectTimes.incrementAndGet());
```
Comment (sandynz) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/client/MySQLClient.java:304]: It's better not put `incrementAndGet` in logging

**Review by sandynz — CHANGES_REQUESTED** [review]

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/client/MySQLClient.java (hunk 1 above)
Comment (terrymanu) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/client/MySQLClient.java:304]: Please rename `mysql` to `MySQL`

Comment (terrymanu) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/client/MySQLClient.java:304]: `retryTimes` are 2 words, please split them

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/client/MySQLClient.java [hunk 2]
```diff
@@ -283,23 +286,23 @@ public void channelInactive(final ChannelHandlerContext ctx) {
...
-            log.error("protocol resolution error", cause);
+            log.error("MySQLBinlogEventHandler: protocol resolution error, lastBinlogEvent={}", lastBinlogEvent, cause);
         }
-        
+    
         private void reconnect() {
-            reconnectTimes++;
-            log.info("reconnect mysql client, retryTimes={}", reconnectTimes);
+            int retryTimes = reconnectTimes.incrementAndGet();
+            log.info("reconnect MySQL client, retry times={}", retryTimes);
```
Comment (sandynz) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/client/MySQLClient.java:305]: The log message is mixed with `:` and `=`, it's better to use unified format.

**Review by sandynz — CHANGES_REQUESTED** [review]

**Review by sandynz — APPROVED** [review]

**Review by sandynz — DISMISSED** [review]

**Comment by azexcy** [comment]
some IT case will reopen after problem is solved

**Review by terrymanu — APPROVED** [review]

**Pull request merged**